src/mono/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -288,6 +285,7 @@
       <Compile Include="$(BclSourcesRoot)\System\Threading\ThreadPoolBoundHandle.Browser.Mono.cs" />
       <Compile Include="$(BclSourcesRoot)\System\Threading\PreAllocatedOverlapped.Browser.Mono.cs" />
       <Compile Include="$(BclSourcesRoot)\System\Threading\ThreadPool.Browser.Mono.cs" />
+      <Compile Include="$(BclSourcesRoot)\System\Threading\TimerQueue.Browser.Mono.cs" />
```

Comment:
Why are we adding it when `FeatureWasmThreads=false`? Aren't we making changes for MT-only and non-MT should stay as it was?